app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_people = [Person(person_data["name"], person_data["age"])
+                      for person_data in people]
+
+    for person_data in people:
+        name = person_data["name"]
+        person = Person.people[name]
+
+        if person_data.get("wife"):
+            wife_name = person_data.get("wife")
+            person.wife = Person.people[wife_name]
+
+        if person_data.get("husband"):
+            husband_name = person_data.get("husband")
```

Comment:
Using new operator, you can do like this
```suggestion
        if wife_name := person_data.get("wife"):
            person.wife = Person.people[wife_name]

        if husband_name := person_data.get("husband"):
```